py-polars/tests/unit/io/test_iceberg.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import annotations
+
+import contextlib
+import os
+from pathlib import Path
+
+import pytest
+
+import polars as pl
+from polars.io.iceberg import _convert_predicate, _to_ast
+
+
+@pytest.fixture()
+def iceberg_path() -> str:
+    # Iceberg requires absolute paths, so we'll symlink
+    # the test table into /tmp/iceberg/t1/
+    Path("/tmp/iceberg").mkdir(parents=True, exist_ok=True)
+    current_path = Path(__file__).parent.resolve()
+
+    with contextlib.suppress(FileExistsError):
+        os.symlink(f"{current_path}/files/iceberg-table", "/tmp/iceberg/t1")
+
+    return "file:///tmp/iceberg/t1/metadata/00001-55cdf97b-255c-4983-b9f3-0e468fadfe9e.metadata.json"
+
+
+def test_scan_iceberg_plain(iceberg_path: str) -> None:
+    df = pl.scan_iceberg(iceberg_path)
+    assert len(df.collect()) == 233822
```

Comment:
Testing the length is OK, but definitely test the schema too (at least in some of the tests).